src/Jobs/CVECheckJob.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Composer security checker job. Runs the task which does the check as a queuedjob.
+ *
+ * @author Peter Thaleikis
+ * @license BSD-3-Clause
+ */
+class CVECheckJob extends AbstractQueuedJob implements QueuedJob
+{
+    private static $dependencies = [
+        'CVECheckTask' => '%$' . CVECheckTask::class,
+    ];
+
+    /**
+     * @var CVECheckTask
+     */
+    protected $checkTask;
+
+    /**
+     * @return CVECheckTask
+     */
+    public function getCVECheckTask()
```

Comment:
Can you please name the getters/setters the same as the property? I.e. `setCheckTask(CheckTask $checkTask)`